cpp/lib/include/FileUtil.h

```diff
@@ -270,6 +270,13 @@ inline std::string MakeAbsolutePath(const char * const relative_path)
 bool MakeEmpty(const std::string &path);
 
 
+/** \brief Create an empty file or change last modified timestamp of an existing file. */
```

Comment:
What about the "last accessed" timestamp?

Reply:
Well to be absolutely correct, touch changes all 3 timestamps (access, modify, change).
https://stuff-things.net/2015/06/17/unix-timestamps-explained/